.github/workflows/release-nightly.yml

```
@@ -9,6 +9,13 @@ concurrency: rel-nightly
 
 jobs:
 
+  show-inputs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          echo "Workflow branch: ${{ github.ref_name }}  " >> $GITHUB_STEP_SUMMARY
```

This can be a single line.
```suggestion
      - run: echo "Workflow branch: ${{ github.ref_name }}  " >> $GITHUB_STEP_SUMMARY
```